MonoHbb/MonoHBranchReader.py

```diff
@@ -811,22 +815,25 @@ def AnalyzeDataSet():
         
         allquantlist=[]
         
-        for region in ['sr','Zcr','Wcr','TOPcr']:                               # Makes all combinations of region, jet number, etc.
+        for region in ['sr','Zcr','Wcr','TOPcr']:                               # Jets: Makes all combinations of region, jet number, etc.
             for jetprop in ['pT','eta','phi','csv']:
                 for jetnum in [1,2]:
                     allquantlist.append('jet'+str(jetnum)+"_"+jetprop+"_"+region+"1")
-                    for lep in ['mu','el']:
-                       if (region != 'Zcr' and jetnum==2 and jetprop == 'csv') or region == 'sr': continue
-                       allquantlist.append(lep+str(jetnum)+"_"+jetprop+"_"+region+"1")
-                       if lep == 'mu':
-                          allquantlist.append(lep+str(jetnum)+"_iso_"+region+"1")
+                    
                 for jetnum in [1,2,3]:
                     allquantlist.append('jet'+str(jetnum)+"_"+jetprop+"_"+region+"2")
-                    for lep in ['mu','el']:
-                       if (region != 'Zcr' and jetnum==2 and jetprop == 'csv') or region == 'sr': continue
-                       allquantlist.append(lep+str(jetnum)+"_"+jetprop+"_"+region+"2")
-                       if lep == 'mu':
-                          allquantlist.append(lep+str(jetnum)+"_iso_"+region+"2")
+                                             
+        
+        for nSR in ['1','2']:                                                   # Leptons: Makes all combinations of region, lepton number, etc.
+            for lep in ['mu','el']:
+                props = ['pT','eta','phi']
+                if lep=='mu':
+                    props.append('csv')  
```

Comment:
Why do you need csv here?

Reply:
Oh, I meant iso! Oops, sorry. Wait, I'll edit and resend.

Reply:
Edited, please review again.